Makefile

```diff
@@ -1,5 +1,16 @@
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMAGE_REGISTRY ?= quay.io
+REGISTRY_NAMESPACE ?= ocs-dev
+IMAGE_TAG ?= latest
+IMAGE_NAME ?= ceph-csi-operator
+BUNDLE_IMAGE_NAME ?= $(IMAGE_NAME)-bundle
+
+# IMG defines the image used for the operator.
+IMG ?= $(IMAGE_REGISTRY)/$(REGISTRY_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG)
```

Comment:
This one also

Reply:
can you please send a PR for this change in upstream?

Reply:
Here it is https://github.com/ceph/ceph-csi-operator/pull/30